pack/return/rttfa.json

```diff
@@ -16,6 +33,134 @@
 		"type_code": "asset",
 		"xp": 2
 	},
+	{
+		"code": "53003",
+		"cost": 1,
+		"deck_limit": 2,
+		"faction_code": "seeker",
+		"flavor": "“How do you know I’m mad?” said Alice.<br/>“You must be,” said the Cat, “or you wouldn’t have come here.”<br/>– Lewis Carroll, Alice’s Adventures in Wonderland",
+		"illustrator": "Maggie Ivy",
+		"name": "Truth from Fiction",
+		"pack_code": "rttfa",
+		"position": 3,
+		"quantity": 2,
+		"skill_intellect": 3,
+		"text": "Place 2 secrets among assets controlled by investigators at your location (3 secrets instead if there is a clue at your location).",
+		"traits": "Insight.",
+		"type_code": "event",
+		"xp": 2
+	},
+	{
+		"code": "53004",
+		"cost": 2,
+		"deck_limit": 2,
+		"faction_code": "seeker",
+		"illustrator": "Greg Bobrowski",
+		"name": "Ancient Stone",
+		"pack_code": "rttfa",
+		"position": 4,
+		"quantity": 2,
+		"skill_agility": 2,
+		"slot": "Hand",
+		"subname": "Transient Thoughts",
+		"text": "You can only include this asset in your deck by upgrading it from Ancient Stone <i>(Unidentified)</i>, and only if you have \"identified the stone\" in your Campaign Log.\nUses (X secrets). X is the number in parentheses next to \"you have identified the stone.\"\n[reaction] When you draw any number of cards, spend that many secrets: Move that many times.",
+		"traits": "Item. Relic.",
+		"type_code": "asset",
+		"xp": 4
+	},	
+	{
+		"code": "53005",
+		"cost": 0,
+		"deck_limit": 2,
+		"faction_code": "rogue",
+		"illustrator": "A.L. Ashbaugh",
+		"name": "Decorated Skull",
+		"pack_code": "rttfa",
+		"position": 5,
+		"quantity": 2,
+		"skill_combat": 1,
+		"skill_willpower": 1,
+		"slot": "Accessory",
+		"subname": "Doom Begets Doom",
+		"text": "Uses (0 charges).\n[reaction] After an investigator, [[Ally]] asset, or enemy at your location is defeated: Place 1 resource <i>(from the token bank)</i> on Decorated Skull, as a charge.\n[action] Spend up to 3 charges: Draw that many cards and gain that many resources.",
+		"traits": "Item. Relic. Cursed.",
+		"type_code": "asset",
+		"xp": 3
+	},
+	{
+		"code": "53006",
+		"cost": 2,
+		"deck_limit": 2,
+		"faction_code": "rogue",
+		"illustrator": "Dual Brush Studios",
+		"name": "Colt Vest Pocket",
+		"pack_code": "rttfa",
+		"position": 6,
+		"quantity": 2,
+		"slot": "Hand",
+		"skill_agility": 1,
+		"skill_combat": 1,
+		"text": "Uses (5 ammo).\n[action] Spend 1 ammo: <b>Fight</b>. You get +2 [combat] for this attack. This attack deals +1 damage.\n<b>Forced</b> - At the end of the round, if you triggered Colt Vest Pocket's \"[action]\" ability: Discard it.",
+		"traits": "Item. Weapon. Firearm. Illicit.",
+		"type_code": "asset",
+		"xp": 2
+	},
+	{
+	  "code": "53007",
```

Comment:
minor nit, but can you tab this section so it matches the rest of the json?

Reply:
Will do!